source/server/core_server.cpp

```diff
@@ -123,6 +127,16 @@ static void RunServer(const ServerConfiguration& config)
   nifgen_grpc::NiFgenService nifgen_service(&nifgen_library, mi_shared_resource_repository);
   builder.RegisterService(&nifgen_service);
 
+  nidaqmx_grpc::NiDAQmxLibrary nidaqmx_library;
+  using DAQmxResourceRepository = nidevice_grpc::SessionResourceRepository<TaskHandle>;
+  auto daq_resource_repository = std::make_shared<DAQmxResourceRepository>(&session_repository);
+  nidaqmx_grpc::NiDAQmxService nidaqmx_service(&nidaqmx_library, daq_resource_repository);
```

Comment:
Stack variables need to be outside the toggle if block.

This code is probably due for a refactor into unique_ptrs, but I think this change is OK without it.